src/screens/exposureHistory/ExposureHistoryScreen.tsx

```diff
@@ -49,17 +49,18 @@ const toOutbreakExposureHistoryData = ({
 };
 
 const toProximityExposureHistoryData = ({
-  proximityExposureTimestamps,
+  proximityExposureHistory,
   i18n,
 }: {
-  proximityExposureTimestamps: number[];
+  proximityExposureHistory: ProximityExposureHistoryItem[];
   i18n: I18n;
 }): CombinedExposureHistoryData[] => {
-  return proximityExposureTimestamps.map(timestamp => {
+  return proximityExposureHistory.map(item => {
     return {
       exposureType: ExposureType.Proximity,
       subtitle: i18n.translate('QRCode.ProximityExposure'),
-      timestamp,
+      timestamp: item.notificationTimestamp,
```

Comment:
Can we use `notificationTimestamp` here?

Reply:
I did this and immediately found a bug! I had been using the exposure timestamp for outbreaks on the history screen.